src/sentry/api/endpoints/organization_release_previous_commits.py

```diff
@@ -44,6 +46,14 @@ def get(self, request, organization, version):
             .extra(order_by=["-date"])[:1]
         )
 
+        analytics.record(
+            "release.get_previous_commits",
```

Comment:
@NisanthanNanthakumar @manuzope Not really sure why we would need to record an analytics event for getting the previous commits. Aren't we really just trying to figure out where releases are coming from? Every call for getting the previous commits would be followed by generating the release as well, yea?

Reply:
This is for knowing who is using local commits cause that hits this endpoint.

Reply:
@manuzope we won't figure that out based on the user agent with the release?

Reply:
Nope. The UA will just tell us if the action is happening via sentry-cli or GA or Bitbucket but won't tell us that it's using `sentry-cli set-commits --local`